src/groupByKey.ts

```diff
@@ -2,6 +2,18 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const Objects: GroupsMap<T> = {};
```

Comment:
this one is quite important from convention perspective
when you capitalize variables they must be `type / interface / react component`
if it's a regular variable - first char must be lower-case
```suggestion
  const objects: GroupsMap<T> = {};
```

Reply:
and ofc apply throughout the code
remember, rename using `F2` to apply new name to all occurences

Reply:
Done